Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.cpp

```diff
@@ -53,19 +53,23 @@ std::unique_ptr<WebXROpaqueFramebuffer> WebXROpaqueFramebuffer::create(PlatformX
 WebXROpaqueFramebuffer::WebXROpaqueFramebuffer(PlatformXR::LayerHandle handle, Ref<WebGLFramebuffer>&& framebuffer, WebGLRenderingContextBase& context, Attributes&& attributes, IntSize framebufferSize)
     : m_handle(handle)
     , m_drawFramebuffer(WTFMove(framebuffer))
-    , m_context(context)
+    , m_context(&context)
     , m_attributes(WTFMove(attributes))
     , m_framebufferSize(framebufferSize)
 {
 }
 
 WebXROpaqueFramebuffer::~WebXROpaqueFramebuffer()
 {
-    if (RefPtr gl = m_context.graphicsContextGL()) {
+    RefPtr context = m_context.get();
+    if (!context)
+        return;
+
+    if (RefPtr gl = context->graphicsContextGL()) {
         m_drawAttachments.release(*gl);
         m_resolveAttachments.release(*gl);
         m_resolvedFBO.release(*gl);
-        m_context.deleteFramebuffer(m_drawFramebuffer.ptr());
+        context->deleteFramebuffer(m_drawFramebuffer.ptr());
```

Comment:
Don't we need a RefPtr / Ref to m_drawFramebuffer on the stack before passing it to deleteFramebuffer()?